src/services/mailer/index.js

```diff
@@ -0,0 +1,30 @@
+import dotenv from 'dotenv';
+import sgMail from '@sendgrid/mail';
+
+dotenv.config();
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+const templates = {
+  verify_email: 'd-1903b0acf59940cea2618e012f891007',
```

Comment:
verif_email id should be stored in `.env` file

Reply:
ok

Reply:
done